### ISSUE TITLE
How to run it on MacOS(m1 chip)?

### ISSUE THREAD

**Issue by Coder7777** [EasyAbp/AbpHelper.GUI#170]
I tried to run it but got an alter. How to fix it? Please advise.

Thanks

<img width="295" alt="image" src="https://github.com/EasyAbp/AbpHelper.GUI/assets/17242425/3eba341a-8f2f-4e7a-963c-753a4fad9f46">

**Comment by Coder7777** [comment]
<img width="507" alt="image" src="https://github.com/EasyAbp/AbpHelper.GUI/assets/17242425/0dca37a7-8516-42a1-a0f0-c2968bfc9953">

**Comment by gdlcf88** [comment]
Try to pull this git repo and build it locally.

**Comment by ryanrocksize5** [comment]
RIght-click on the application and click open , you'll have to do this twice for the app to actually open.
